kong/db/schema/entities/routes.lua

```diff
@@ -195,27 +177,26 @@ else
           },
         },
       } },
-      { https_redirect_status_code = { type = "integer",
-                                       description = "The status code Kong responds with when all properties of a Route match except the protocol",
-                                       one_of = { 426, 301, 302, 307, 308 },
-                                       default = 426, required = true,
-                                     }, },
+
       { regex_priority = { description = "A number used to choose which route resolves a given request when several routes match it using regexes simultaneously.", type = "integer", default = 0 }, },
-      { strip_path     = { description = "When matching a Route via one of the paths, strip the matching prefix from the upstream request URL.", type = "boolean", required = true, default = true }, },
       { path_handling  = { description = "Controls how the Service path, Route path and requested path are combined when sending a request to the upstream.", type = "string", default = "v0", one_of = { "v0", "v1" }, }, },
-      { preserve_host  = { description = "When matching a Route via one of the hosts domain names, use the request Host header in the upstream request headers.", type = "boolean", required = true, default = false }, },
-      { request_buffering  = { description = "Whether to enable request body buffering or not. With HTTP 1.1.", type = "boolean", required = true, default = true }, },
-      { response_buffering  = { description = "Whether to enable response body buffering or not.", type = "boolean", required = true, default = true }, },
-      { snis = { type = "set",
-                 description = "A list of SNIs that match this Route when using stream routing.",
-                 elements = typedefs.sni }, },
-      { sources = typedefs.sources },
-      { destinations = typedefs.destinations },
-      { tags             = typedefs.tags },
-      { service = { description = "The Service this Route is associated to. This is where the Route proxies traffic to.",
-      type = "foreign", reference = "services" }, },
-    },
+    },  -- fields
 
     entity_checks = entity_checks,
+} -- routes
+
+
+if kong_router_flavor == "expressions" then
+
+  local special_fields = {
```

Comment:
seems that we could call `table.insert` on these two array into `routes.field` directly.

If we still need to expand these special_fields in the future, just ignore my comment.

Reply:
Currently the only needed field is `expression`, we can change the code if we have more new fields in the future.

Reply:
cool, so let we use special_fields + for loop for potential extented fields